source/use-cases/client-side-field-level-encryption-guide.txt

```diff
@@ -250,15 +262,37 @@ information on the features of envelope encryption and key management
 concepts, see :aws-docs:`AWS Key Management Service Concepts
 <kms/latest/developerguide/concepts.html#enveloping>`.
 
-The master key, used by the MongoDB driver to create and encrypt data
-keys, should be stored remotely in a :wikipedia:`Key Management System
-<Key_management#Key_management_system>`. The data encryption keys,
-generated and used by the MongoDB driver to encrypt and decrypt document
-fields, are stored in a key vault collection in the same database as the
-encrypted data.
+The following diagram shows how the **master key** and **data encryption
+keys** are created and stored in the MongoDB implementation of CSFLE:
+
+.. image:: /figures/CSFLE_Create_Keys_Local.png
+   :alt: Diagram that describes creating master and data encryption keys when using a local provider
```

Comment:
This image does not render on the staged docs for me

![image](https://user-images.githubusercontent.com/15657698/70574570-6a306680-1b72-11ea-98b2-a9a719c271ce.png)

Reply:
Upon further review, NONE of the images render for me

Reply:
Will try to figure out how to make it work.

Reply:
The images work now.